pkg/resources/fluentbit/daemonset.go

```diff
@@ -147,6 +154,44 @@ func (r *Reconciler) generatePortsMetrics() (containerPorts []corev1.ContainerPo
 	return
 }
 
+func newConfigMapReloader(spec *v1beta1.FluentbitSpec) corev1.Container {
+	var args []string
+	vm := []corev1.VolumeMount{
+		{
+			Name:      "config",
+			MountPath: OperatorConfigPath,
+		},
+	}
+
+	args = append(args,
+		"--volume-dir=/fluentbit/config",
```

Comment:
shouldn't this be `OperatorConfigPath`?

Reply:
@aslafy-z are you sure this is the right path?

Reply:
Nice catch. It's still untested!